spark/v3.0/spark/src/main/java/org/apache/iceberg/spark/actions/BaseMigrateTableSparkAction.java

```diff
@@ -95,6 +97,12 @@ public MigrateTable tableProperty(String property, String value) {
     return this;
   }
 
+  @Override
+  public MigrateTable dropBackup(boolean dropBackup) {
+    this.removeBackup = dropBackup;
```

Comment:
Can we rename the variables to be the same (either remove or drop).  It makes the code easier if there's fewer concepts, and we keep the same variable name.

Reply:
Done. Fixed in the latest commit.

Reply:
OK sorry, i guess it breaks the checkstyle :(.   

After taking a look through other Iceberg interfaces, I think nobody passes a boolean but just have a no-arg method.  Can we do that instead?  Thanks.

ie. 
dropBackup() {
   this.dropBackup = true;
   return this;
}

Reply:
I fixed the checkstyle issue in the latest commit. Thanks for your patience in reviewing this PR.